Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1029,7 +1028,7 @@ def add_replace_strs(self, *args):
             Add strings which will be replaced when logging.
             Meant for avoiding passwords and so forth in the log.
         '''
-        to_add = [self.encode(a) for a in args if a]
+        to_add = [stringUnEscape(self.encode(a)) for a in args if a]
```

Comment:
should add both versions:
* stringUnEscape(self.encode(a))
* self.encode(a)

Reply:
You need to add both. But you need to add the opposite of stringUnEscape. Something like `stringEscape`